### DISCUSSION TITLE
[Request] Navidrome Music Server

### DISCUSSION THREAD

**Discussion by moppedz** [tteck/Proxmox#478] [Ideas]
Local media server/streamer with subsonic clients support on android, ios.
Pretty much the closest to spotify i have seen. 
Used Jellyfin and several others before, but they are a pain in the ass when it comes to music collections
and/or offline support.

https://github.com/navidrome/navidrome/
https://www.navidrome.org/

**Comment by tteck** [comment]
Done!

**Comment by moppedz** [comment]
Noice! Thank you!
